packages/app/src/app/vscode/initializers.ts

```diff
@@ -3,9 +3,12 @@ import { vscode } from 'app/vscode';
 export function initializeThemeCache() {
   try {
     if (!localStorage.getItem('vs-global://colorThemeData')) {
+      localStorage.setItem('newUser', 'true');
```

Comment:
I think we shouldn't put the `newUser` code in the `vscode` initializers, we don't use this code in some cases and new user is unrelated to whether VSCode initialized or not. Let's find a different property that we can use for this, I think there are easier ways to check this, by eg. checking the `signed_in` cookie. cc @SaraVieira